components/Mentor/structure.ts

```
@@ -9,41 +9,57 @@ export const getValidationSchema = (hasSubareas, requiresDate) => {
-    indicationLinkedin: Yup.string()
-      .when('indication', {
-        is: 'sim',
-        then: (schema) => schema.matches(/((https:\/\/)((www|\w\w)\.)linkedin\.com\/)((([\w]{2,3}))|([^\/]+\/(([\w|\d-&#=])+\/){1,}))$/, "Linkedin inválido.")
+      .matches(
+        /^(https?:\/\/)(www\.)?linkedin\.com\/in\/[\w-]{3,}(\/)?(\?.*)?(#.*)?$/,
+        "Por favor, insira um link válido do LinkedIn (ex.: https://www.linkedin.com/in/seu-nome)."
+      )
+      .required("O campo LinkedIn é obrigatório."),
+    indicationLinkedin: Yup.string().when("indication", {
+      is: "sim",
+      then: (schema) =>
```

Biome (2.1.2)

[error] Do not add then to an object.



(lint/suspicious/noThenProperty)